fxgl-trade/src/main/kotlin/com/almasb/fxgl/inventory/view/InventoryView.kt

```diff
@@ -3,16 +3,40 @@
  * Copyright (c) AlmasB (almaslvl@gmail.com).
  * See LICENSE for details.
  */
-
 package com.almasb.fxgl.inventory.view
 
 import com.almasb.fxgl.inventory.Inventory
-import javafx.scene.Parent
+import com.almasb.fxgl.inventory.ItemData
+import javafx.collections.ObservableList
+import javafx.scene.layout.Pane
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+import javafx.scene.shape.Rectangle
 
 /**
  *
  * @author Almas Baimagambetov (almaslvl@gmail.com)
  */
-class InventoryView<T>(val inventory: Inventory<T>) : Parent() {
+class InventoryView<T>(val inventory: Inventory<T>, width: Int, height: Int) : Pane(){
```

Comment:
This is related to #957 so can potentially close two issues at once. Would it be possible to redesign the view to not use css? Just a default ListView should do it I think